mp/src/game/client/momentum/ui/HUD/hud_speedometer.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "hudelement.h"
+
+#include <vgui_controls/EditablePanel.h>
+
+#define SPEEDOMETER_MAX_LABELS 4
+
+namespace vgui 
+{
+    class Label;
+}
```

Comment:
Is this needed? You don't seem to use any labels in this class, only SpeedometerLabel.

Reply:
Another oops. Removed this.